packages/protocol/contracts/common/InitializableV2.sol

```diff
@@ -1,21 +0,0 @@
-pragma solidity ^0.5.13;
-
-contract InitializableV2 {
-  bool public initialized;
-
-  constructor(bool testingDeployment) public {
-    if (!testingDeployment) {
-      _initialize();
-    }
-  }
-
-  modifier initializer() {
-    require(!initialized, "contract already initialized");
-    _initialize();
-    _;
-  }
-
-  function _initialize() private {
```

Comment:
Ahh I wonder if this helper was what caused the bytecode to change when I first tried it